h2/src/main/org/h2/mvstore/FreeSpaceBitSet.java

```diff
@@ -239,6 +242,24 @@ long getLastFree() {
         return getPos(set.previousSetBit(set.size()-1) + 1);
     }
 
+    int getMovePiority(int block) {
```

Comment:
would be nice to have the comment you made for this PR here, so it's easy to see at a glance what it is doing

Reply:
That was my thought too, the second after I pushed the button. :smile: